src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/article/Entity/OverviewPage.php

```diff
@@ -36,16 +36,6 @@ public function getPageTemplates()
 	return array('{{ bundle.getName() }}:{{ entity_class|lower }}overviewpage');
     }
 
-    /**
-     * @param ContainerInterface $container
-     * @param Request            $request
-     * @param RenderContext      $context
-     */
-    public function service(ContainerInterface $container, Request $request, RenderContext $context)
```

Comment:
Are you sure this doesn't break the article overview pages? I don't see a SlugActionInterface in this class or it's parent (AbstractArticleOverviewPage).